x-pack/solutions/observability/plugins/slo/public/hooks/use_fetch_slo_health.ts

```diff
@@ -18,29 +18,34 @@ export interface UseFetchSloHealth {
 }
 
 export interface Params {
-  list: SLOWithSummaryResponse[];
+  page?: number;
+  perPage?: number;
+  statusFilter?: 'healthy' | 'unhealthy';
+  list?: SLOWithSummaryResponse[];
 }
 
-export function useFetchSloHealth({ list }: Params): UseFetchSloHealth {
+export function useFetchSloHealth({ list, page, perPage }: Params): UseFetchSloHealth {
   const { sloClient } = usePluginContext();
-  const payload = list.map((slo) => ({
-    sloId: slo.id,
-    sloInstanceId: slo.instanceId ?? ALL_VALUE,
-  }));
+  const payload = list
+    ? list.map((slo) => ({
+        sloId: slo.id,
+        sloInstanceId: slo.instanceId ?? ALL_VALUE,
+      }))
+    : [];
 
   const { isLoading, isError, data } = useQuery({
     queryKey: sloKeys.health(payload),
     queryFn: async ({ signal }) => {
       try {
         return await sloClient.fetch('POST /internal/observability/slos/_health', {
-          params: { body: { list: payload } },
+          params: { body: { list: payload, page, perPage, statusFilter: 'unhealthy' } },
           signal,
         });
       } catch (error) {
         // ignore error
       }
     },
-    enabled: Boolean(list.length > 0),
+    enabled: true,
```

Comment:
The `enabled` property controls whether the query should automatically run. You set it here always to true. Why can't we keep it as it was before? Any specific reason to remove that check of list length?